static/js/multi-level-switch.js

```diff
@@ -23,18 +23,20 @@ const MULTI_LEVEL_SWITCH_ON_BLANK = 'white';
  * @param Object description Thing description object.
  * @param {String} format 'svg', 'html', or 'htmlDetail'.
  */
-function MultiLevelSwitch(description, format) {
+function MultiLevelSwitch(description, format, options) {
   if (format === 'htmlDetail') {
     this.details = this.details || {};
     this.details.on = new OnOffDetail(this);
     this.details.level  = new LevelDetail(this);
   }
 
+  const opts = options || {svgBaseIcon: '/images/level.svg',
+                           pngBaseIcon: '/images/level.svg',
```

Comment:
In the future I think it would be good to have more descriptive names than `svgBaseIcon` and `pngBaseIcon` since the actual distinction is where they appear in the UI